include/text.php

```diff
@@ -933,12 +933,12 @@ function contact_block() {
 	$a = get_app();
 
 	$shown = get_pconfig($a->profile['uid'],'system','display_friend_count');
-	if($shown === false)
+	if ($shown === false)
```

Comment:
Here are some more missing brackets. Do you want to rewrote in several passes or is it planned to be perfect with the first one?